src/main/java/com/internetitem/logback/elasticsearch/writer/ElasticsearchWriter.java

```diff
@@ -16,13 +16,15 @@
 
 public class ElasticsearchWriter implements SafeWriter {
 
+	private StringBuilder backupBuffer;
```

Comment:
THis buffer holds the UPCOMING request if the current pending request is already full.